### ISSUE TITLE
Sanity Check for (Optional) Fragments Section Uses Wrong Code

### ISSUE THREAD

**Issue by mrk-han** [udacity/ud839_Miwok#78]
https://raw.githubusercontent.com/udacity/ud839_Miwok/d7effcef3bf7fdccdd045c974d67abd5b960fae7/app/src/main/java/com/example/android/miwok/NumbersFragment.java

```
        words.add(new Word(R.string.number_one, R.string.miwok_number_one,
                R.drawable.number_one, R.raw.number_one));
```

I'm pretty confident that we were never told to put the string "lutti" into our values/strings.xml file as miwok_number_one.  So, if we copy and paste this code into our project we have to go through and add all the strings to our strings.xml file for it to compile.  

My code looks like this:
```
        words.add(new Word("one", "lutti", R.drawable.number_one, R.raw.number_one));
```

Also, if you go to lesson 5 branch of the main Miwok Repo, it won't even compile because
```
R.string.miwok_number_one
```
Does not exist.

TL;DR:
```
R.string.miwok_number_one
```
is missing from the `values/strings.xml` file in the branch `lesson-five`

And, the code shown in the sanity check in the URL at the top of this issue does not match what the user would have unless they took all of their strings and put them in the `values/strings.xml` file which I'm not positive we were told to do yet.

**Comment by SudKul** [comment]
Please submit a PR should you have any suggestions for improvement.
